eng/config/rulesets/Shipping.ruleset

```diff
@@ -110,6 +110,7 @@
     <Rule Id="RS0026" Action="Error" />
     <Rule Id="RS0027" Action="Error" />
     <Rule Id="RS0033" Action="None" /> <!-- Importing constructor should be [Obsolete] -->
+    <Rule Id="RS0040" Action="None" /> <!-- Defaultable types should have defaultable fields -->
```

Comment:
📝 Enabling this is high-churn. I'm going to submit it as a separate pull request.